lib/lists.nix

```diff
@@ -73,7 +73,7 @@ rec {
        lconcat [ "a" "b" "c" ]
        => "zabc"
        # different types
-       lstrange = foldl (str: int: str + toString (int + 1)) ""
+       lstrange = foldl (str: int: str + toString (int + 1)) "a"
        strange [ 1 2 3 4 ]
```

Comment:
When we're already at it, this should be `lstrange` here, right?

Reply:
(Doesn't appear to be done yet, maybe @drewmullen forgot a push?)

Reply:
sorry about that... must have borked the push and not noticed!